src/pages/iou/request/step/IOURequestStepTaxAmountPage.js

```diff
@@ -82,7 +82,7 @@ function IOURequestStepTaxAmountPage({
     );
 
     const navigateBack = () => {
-        Navigation.goBack(isEditing ? ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, reportID) : ROUTES.HOME);
+        Navigation.goBack(backTo || ROUTES.HOME);
```

Comment:
You need to define backTo?

Reply:
O yea backTo is defined and comes from route params.

Reply:
Right, but don't we need something like - 
`const backTo = lodashGet(route, 'params.backTo', ROUTES.HOME);`

Reply:
no, we don't, issues with fallback is already handled [here](https://github.com/teneeto/Expensify/blob/72693dc25bdf4e345026933e21273d9df901e250/src/ROUTES.ts#L9)

Reply:
again, we have other scenarios were backTo is refactored to be used this way.. we could maintain that and avoid one more variable. what do you think?

Reply:
https://github.com/Expensify/App/blob/72693dc25bdf4e345026933e21273d9df901e250/src/pages/iou/request/step/IOURequestStepTaxAmountPage.js#L56
I think everything looks good here